### PR TITLE
Doc/faq and blog doc with prismic

### PR DESCRIPTION
**Pull request by MarieTheCoin**
1-Home
2-FAQType
3-ArticleType
4-Categories

### CODE REVIEW

File: apps/site-landing/src/components/Prismic/types.ts [hunk 1]
```diff
@@ -32,7 +32,8 @@ export type ArticleResult = {
   thumbnail: ImageObj,
   image_before_title: ImageObj,
   title: RenderableType[]|null;
-  questions: RenderableType[]|null;
+  publication_date: string; 
+  short_content: RenderableType[]|null;
```
Comment (MarieTheCoin) [inline apps/site-landing/src/components/Prismic/types.ts:36]: Forgot those 2 fields

File: apps/site-landing/src/containers/Blog/Article.tsx [hunk 2]
```diff
@@ -32,15 +33,16 @@ function getTranslatedArticle(filtered:ArticleDocument[],docs:ArticleDocument[])
...
+  const { locale } = useSelector(selectLocale);
   let history = useHistory();
-
   return <>{
     filtered.map(articleData => (    
       <div className={styles.containerArticle} key={articleData.id}>
         <div className={` ${styles.backLink} x6spaceBefore`}><a onClick={() => history.goBack()}><Icon name="arrow left"/><FormattedMessage {...backLink} /></a></div>
         { articleData.data.image_before_title.url ? <img src={ articleData.data.image_before_title.url} alt={articleData.data.image_before_title.alt}/> : "" }
         { <Header as={"h2"} className="x6spaceBefore">{articleData.data.title ? articleData.data.title[0].text : ""}</Header> }
+        <p>{ DateTime.fromFormat(articleData.data.publication_date, "yyyy-mm-dd", { }).setLocale(locale).toLocaleString(DateTime.DATE_HUGE)}</p>
```
Comment (MarieTheCoin) [inline apps/site-landing/src/containers/Blog/Article.tsx:45]: Added the article date on the article page

File: apps/site-landing/src/containers/Blog/ArticleItem.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const ArticleItem = ({ id, data }: ArticleDocument) => {
           <Grid.Column>
             <div className={`${styles.text}`}>
               <Header as={"h4"}>{data.title ? data.title[0].text : ""}</Header>
-              { RichText.render(data.content as unknown as RichTextBlock[]) }
+              { RichText.render(data.short_content as unknown as RichTextBlock[]) }
```
Comment (MarieTheCoin) [inline apps/site-landing/src/containers/Blog/ArticleItem.tsx:25]: Changed the content to the short content field for the list article pages

**Comment by FrozenKiwi** [comment]
Seems fairly comprehensible - only problem with text in the images is it's not searchable.  Any chance you could add some keywords to the `md` document, so it turns up in searchs?  I'm not sure which ones make sense - probably at least prismic/category/translation (etc).

Documentation is easy to forget, so being searchable is a good reminder

**Comment by FrozenKiwi** [comment]
Doesnt' really require explanation, but when your searching in code it's quite reassuring to see an `md` file turn up

**Review by FrozenKiwi — APPROVED** [review]

**Pull request merged**